ecscollector/collector.go

```diff
@@ -29,6 +29,21 @@ import (
 const nanoSeconds = 1.0e9
 
 var (
+	metadataDesc = prometheus.NewDesc(
+		"ecs_metadata",
```

Comment:
Minor nit, `ecs_metadata_info` to indicate this is an info metric.

Reply:
done